cafebabel/templates/home.html

```diff
@@ -39,4 +39,30 @@ <h2>Recently published</h2>
     {{ card(article) }}
   {% endfor %}
 {% endif %}
+  <div id=under-construction>
+    <div class=modal>
+    <button></button>
+      <h2>Welcome to the brand new Cafébabel!</h2>
+      <p>
+        We're still under construction.<img src={{ url_for('static', filename='img/emoji/emoji-hammer.png') }} alt="hammer emoji"><br>
+        But if you want to take a look <img src={{ url_for('static', filename='img/emoji/emoji-eyes.png') }} alt="eyes emoji"> around
+        then feel free to login and send us your feedback here:<br>
+        <a href=mailto:webmaster@cafebabel.com>webmaster@cafebabel.com</a>
```

Comment:
bewarn that you have a very high chance that this mailbox gets spammed

Reply:
aie, you are right. How can I do then?

Reply:
https://stackoverflow.com/questions/748780/best-way-to-obfuscate-an-e-mail-address-on-a-website